schemas/health/cy/cris_0001.json

```diff
@@ -707,7 +707,7 @@
                                 "title": "Cafwyd problem wrth agor yr arolwg",
                                 "description": [
                                     {
-                                        "text": "<p>Mae\u2019r holiadur hwn ar gyfer {first_name}. Rydych chi wedi nodi nad chi yw {first_name} ac nad ydych yn cwblhau\u2019r holiadur ar ei ran/rhan. </p><p>Os yw hyn yn gywir:</p><p>1.Ticiwch y blwch parhau ac yna dewiswch &#x201C;Cadw a pharhau&#x201D;.</p><p>2.Ar y dudalen nesaf, dewiswch y botwm &#x201C;Cyflwyno&#x201D; i gyflwyno\u2019r holiadur.</p><p>3.Ffoniwch y llinell gymorth ar 0xxxxxxxxxx. Bydd angen eich Rhif Adnabod Cyfranogwr arnoch. Bydd yr aelod o d\u00eem yr arolwg sydd ar y ff\u00f4n yn dweud wrthych beth i\u2019w wneud nesaf.</p><p><em>Os mai chi yw {first_name}</em>, neu os ydych chi\u2019n ateb y cwestiynau ar ran yr unigolyn:</p><p>1.Dewiswch <em>&#x201C;&lt; Blaenorol&#x201D;</em> ddwywaith. Gallwch ddod o hyd iddo ar frig y dudalen hon <em>ar yr ochr chwith</em>, o dan y faner las sy\u2019n dweud &#x201C;Arolwg COVID-19 a Heintiau Anadlol&#x201D;.</p><p>2.Yna bydd angen i chi ddiwygio\u2019r atebion er mwyn gallu cwblhau\u2019r arolwg.</p>",
+                                        "text": "<p>Mae\u2019r holiadur hwn ar gyfer {first_name}. Rydych chi wedi nodi nad chi yw {first_name} ac nad ydych yn cwblhau\u2019r holiadur ar ei ran/rhan. </p><p>Os yw hyn yn gywir:</p><p>1.Ticiwch y blwch parhau ac yna dewiswch &#x201C;Cadw a pharhau&#x201D;.</p><p>2.Ar y dudalen nesaf, dewiswch y botwm &#x201C;Cyflwyno&#x201D; i gyflwyno\u2019r holiadur.</p><p>3.Ffoniwch y llinell gymorth ar 0808 196 1270. Bydd angen eich Rhif Adnabod Cyfranogwr arnoch. Bydd yr aelod o d\u00eem yr arolwg sydd ar y ff\u00f4n yn dweud wrthych beth i\u2019w wneud nesaf.</p><p><em>Os mai chi yw {first_name}</em>, neu os ydych chi\u2019n ateb y cwestiynau ar ran yr unigolyn:</p><p>1.Dewiswch <em>&#x201C;&lt; Blaenorol&#x201D;</em> ddwywaith. Gallwch ddod o hyd iddo ar frig y dudalen hon <em>ar yr ochr chwith</em>, o dan y faner las sy\u2019n dweud &#x201C;Arolwg COVID-19 a Heintiau Anadlol&#x201D;.</p><p>2.Yna bydd angen i chi ddiwygio\u2019r atebion er mwyn gallu cwblhau\u2019r arolwg.</p>",
```

Comment:
I checked the .po file and it seems like it's not matching what's in there:  `"<p>Mae'r holiadur hwn ar gyfer {first_name}. Rydych chi wedi nodi nad chi yw {first_name} ac nad ydych yn cwblhau'r holiadur ar ei ran/rhan. </p><p>Os yw hyn yn gywir:</p><p>1.Ticiwch y blwch parhau ac yna dewiswch &#x201C;Cadw a pharhau&#x201D;.</p><p>2.Ar y dudalen nesaf, dewiswch y botwm &#x201C;Cyflwyno&#x201D; i gyflwyno'r holiadur.</p><p>3.Ffoniwch y llinell gymorth ar 0808 196 1270. Bydd angen eich Rhif Adnabod Cyfranogwr arnoch. Bydd yr aelod o dîm yr arolwg sydd ar y ffôn yn dweud wrthych beth i'w wneud nesaf.</p><p><em>Os mai chi yw {first_name}</em>, neu os ydych chi'n ateb y cwestiynau ar ran yr unigolyn:</p><p>1.Dewiswch <em>&#x201C;&lt; Blaenorol&#x201D;</em> ddwywaith. Gallwch ddod o hyd iddo ar frig y dudalen hon <em>ar yr ochr chwith</em>, o dan y faner las sy'n dweud &#x201C;Arolwg COVID-19 a Heintiau Anadlol&#x201D;.</p><p>2.Yna bydd angen i chi ddiwygio'r atebion er mwyn gallu cwblhau'r arolwg.</p>"`

Reply:
The only diff should be the apostrophe, that is because the translation process converts the apostrophes to unicode. They use diff encodings. It's an existing issue that hasn't been resolved. It is not a functional issue as it will render the same.